example/config/index.html

```diff
@@ -4,8 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="bloggista.css">
-    <link rel="stylesheet" href="custom.css">
+    <link rel="stylesheet" href="./custom.css">
```

Comment:
Can you change this to be like this (no . (dot))?
```suggestion
    <link rel="stylesheet" href="/custom.css">
```
Not that I am deploying example, but it still wrong. With the dot, the file is going to be look up from the current route. Without it, it will look for the root folder (which is where Bloggista keeps it).